superset-frontend/src/components/Chart/ChartContextMenu.tsx

```diff
@@ -41,6 +41,7 @@ export interface Ref {
     filters: QueryObjectFilterClause[],
     clientX: number,
     clientY: number,
+    hasVizPluginSupport: boolean,
```

Comment:
I suggest adding a flag on the chart's metadata instead of using a property, while the flag in the metadata refers to [this](https://github.com/apache/superset/blob/875e9f8a04464b0e79cfbda21d1e1ac7895970a5/superset-frontend/packages/superset-ui-core/src/chart/models/ChartMetadata.ts#L31).

Reply:
Done, thanks!